hub/management/commands/import_flood_risk_data.py

```diff
@@ -0,0 +1,107 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaData, DataSet, DataType
+
+
+class Command(BaseCommand):
+    help = "Import flood risk data"
+
+    data_file = settings.BASE_DIR / "data" / "risk_of_flooding.csv"
+    message = "Importing constituency flood risk data"
+    defaults = {
+        "label": "Flood risk from rivers or sea",
+        "description": "Data relating to risk of flooding from rivers or sea, recorded in 2018",
```

Comment:
probably worth adding something to these to make it clear it's for england only, maybe just add "in England" after sea, or "for England" after Data.